client/components/theme/style.scss

```diff
@@ -79,20 +79,37 @@ $theme-info-height: 54px;
 	}
 }
 
-.theme-badge__price {
+.theme__badge-price {
 	flex: 0 0 auto;
 	padding: 18px 10px 0;
 	color: $alert-green;
 	font-size: 13px;
 	font-weight: 600;
 }
 
-.theme-badge__price-upgrade {
+.theme__upsell {
+	flex: 0 0 auto;
+	padding: 16px 10px 0 0;
+	color: $gray;
+}
+
+.theme__upsell-heading {
+	text-align: center
+}
+
+.theme__upsell-cta {
+	margin-top: 10px;
+	margin-left: auto;
+	margin-right: auto;
+	display: table;
```

Comment:
It seems weird that this is set to `display: table`.

If you could add a class to `.popover__inner` and apply `text-align: center` to it, you could get rid of:

`.theme__upsell-heading {
  text-align: center
}`

and 

`margin-left: auto;
margin-right: auto;
display: table;`

under `.theme__upsell-cta`